### ISSUE TITLE
Add option to suppress errors

### ISSUE THREAD

**Issue by robrecord** [Rigellute/spotify-tui#1067]
Hi there,

The service seems to work fine but every now and then I get a 503 which takes over the display.

Is it possible to either:

a) Only show this screen when the number of errors is above a (user defined) threshold,
b) have the option of showing it as an indicator or status bar instead,
c) perhaps keep trying in the background and remove message when resolved?

**Comment by ForeignExchangeZtudent** [comment]
Yeah, unfortunately this was one of the big issues with spotify-tui that I imagined would become a problem in this state while I do think showing Error information is important in a lot of applications. 

With Spotify-tui it was never really useful and even in the circumstance that the application was having catastrophical failure. The savvy person would understand to just add error handling back via config if necessary.

I genuinely believe the best outcome would be to just add it as a configuration setting whether more user defined as you're suggesting or just a default on with the option of configuring it off.

Cheers and have a nice weekend.

**Comment by sewnie** [comment]
use https://github.com/aome510/spotify-player
